test/drawing_test.dart

```diff
@@ -3,11 +3,39 @@ import "dart:io";
 import "package:dart_mappable/dart_mappable.dart";
 import "package:flutter/material.dart";
 import "package:flutter_gyw/flutter_gyw.dart";
+import "package:flutter_gyw/src/custom_mappers.dart";
 import "package:flutter_test/flutter_test.dart";
 
+const fontMapper = GYWFontMapper();
+const iconMapper = GYWIconMapper();
+
 void main() {
   group("JSON", () {
-    test("Text without optional values", () {
+    test("Built-in font", () {
+      final GYWFont font = GYWFonts.robotoMonoBold.font;
+      expect(fontMapper.decode(fontMapper.encode(font)), font);
+    });
+
+    test("Custom font", () {
+      const GYWFont font = GYWFont(filename: "custo");
+      // Unknown fonts are replaced by the default one.
+      expect(
+        fontMapper.decode(fontMapper.encode(font)),
+        GYWFonts.robotoMono.font,
+      );
+    });
+
+    test("Built-in icon", () {
+      final GYWIcon icon = GYWIcons.nfc.icon;
+      expect(iconMapper.decode(iconMapper.encode(icon)), icon);
+    });
+
+    test("Custom icon", () {
+      const GYWIcon icon = GYWIcon(name: "custom", filename: "custom");
+      expect(iconMapper.decode(iconMapper.encode(icon)), icon);
+    });
```

Comment:
I think you're gonna hate me for changing mind, but considering that FontMapper are not part of the public API, they should not be used in the tests scripts. So even if it was a good implementation and a good idea, we can remove these

Reply:
No problem, I'm not judging :laughing:.

I can remove them, or maybe I can make GYWIcon and GYWFont be MappableClasses, which add a public toJson. I didn't do it initially because we have some some custom operator== and hashCode that only take filename and name into the equation ignoring the other fields.

Reply:
Speaking of operator== and hashCode, they are inconsistent on GYWFont, because operator== takes into account only filename while hashCode does filename _and_ name. Should we take all fields into account: filename, name, and all the optional ones?

Reply:
If yes and we do it like for GYWIcon, then maybe it's worth annotating them with `@MappableClasses` and have them generated automatically.

Reply:
For `GYWFont` (and `GYWIcon`), the only thing that matters on the device side is the filename. The rest is just some kind of sprinkles and used in the apps for visualizations. Therefore, hashCode and equalities must only take `filename` into account.